charts/lfx-v2-project-service/templates/ingressroute.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright The Linux Foundation and each contributor to LFX.
+# SPDX-License-Identifier: MIT
+---
+# TODO: use the newer Gateway API instead of a IngressRoute resource
+# https://doc.traefik.io/traefik/routing/providers/kubernetes-gateway/
+apiVersion: traefik.io/v1alpha1
+kind: IngressRoute
+metadata:
+  name: lfx-v2-project-service
+  namespace: lfx
+spec:
+  entryPoints:
+    - web
+    - websecure
+  routes:
+    - kind: Rule
+      match: >-
+        Host(`{{.Values.ingress.hostname}}`) &&
+        (Path(`/projects`) || PathPrefix(`/projects/`) || Path(`/livez`) || Path(`/readyz`))
```

Comment:
`/livez` and `/readyz` don't need to be routed to as they're used internally by Kubernetes.